scripts/stg-env.js

```diff
@@ -6,8 +6,8 @@ fs.readFile(envConfig, 'utf8', function (err, data) {
         return console.log(err);
     }
     const result = data
-        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://volta-identitycache.energyweb.org/v1\'')
-        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'https://volta-identityevents.energyweb.org/\'')
+        .replace('cacheServerUrl = \'https://identitycache-dev.energyweb.org/v1\'', 'cacheServerUrl = \'https://identitycache-staging.energyweb.org//v1\'')
+        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'wss://identityevents-staging.energyweb.org\'')
```

Comment:
Are you pretty sure this change to `wss` from `https` is correct?

Reply:
Nope, i used value the you provided. I can check it.

Reply:
Ah haha, yea, probably. Idk, I think better to use `https` even though it's weird and then change all the config to `wss` later
```suggestion
        .replace('natsServerUrl = \'https://identityevents-dev.energyweb.org/\'', 'natsServerUrl = \'https://identityevents-staging.energyweb.org\'')
```